src/diffusers/image_processor.py

```diff
@@ -251,3 +251,115 @@ def postprocess(
 
         if output_type == "pil":
             return self.numpy_to_pil(image)
+
+
+class VaeImageProcessorLDM3D(VaeImageProcessor):
+    """
+    Image Processor for VAE LDM3D
```

Comment:
@patrickvonplaten @sayakpaul maybe we can just refactor the one we have?

Reply:
@estelleafl please also add an entry here:
https://github.com/huggingface/diffusers/blob/main/docs/source/en/api/image_processor.mdx

So that this new image processor properly reflects in the docs.

Reply:
Done!